### PR TITLE
[DinoMod] Mutations, nests, and laser dinos

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary

SUMMARY: Mods "DinoMod content and bugfix rollup"

#### Purpose of change

Catch BN up to DDA 0.F DinoMod content and fixes

#### Describe the solution

Intended to provide the last four months of content and fixes. Included NPC mission fixes and new mutations content and spawn updates and all changes to root directory.  New nests. New DinoLab Finale map. New dinos. BN first: Hatchling content QA, zombie dino upgrade reorg, item content cleanup, CBM zombie reorg.

#### Additional context

"Were dinosaurs fatter and weirder than we suspect?"

### CODE REVIEW

File: data/mods/DinoMod/items/mutagen.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    "id": "iv_mutagen_tyrant",
+    "copy-from": "iv_mutagen_flavor",
+    "type": "COMESTIBLE",
+    "name": "tyrant serum",
+    "description": "A super-concentrated mutagen strongly resembling children's toy slime.  You need a syringe to inject it… if you really want to?",
+    "price": 1000000,
+    "color": "light_green",
+    "healthy": -4,
+    "use_action": { "type": "mutagen_iv", "mutation_category": "  TYRANT" }
```
Comment (Coolthulhu) [inline data/mods/DinoMod/items/mutagen.json:11]: Category has 2 extra spaces

**Comment by LyleSY** [comment]
I've fixed all the Travis errors I understand, but I think some of the new maps and items are creating some new problems I don't get.

**Comment by Coolthulhu** [comment]
> Invalid body part legacy id arm_l

At the moment body parts have to be written in ALL_CAPS.
I'll try to add a hack that will enable using no_caps body parts.

> Failed to visit member bullet in JsonObject

Change all "bullet" to "stab".

> invalid furniture id "f_machinery_electronic"

There is no such furniture in BN. You can either add it to the mod or replace the furniture.

> line 34:38: no such monster

```
"monster":

                                     ^

                                      "mon_maiasaura"
```

It looks like a monster from the mod. You probably didn't add it or misspelled its id.

**Comment by Coolthulhu** [comment]
I added a workaround for the body part thing, so you probably won't need to rename all the body parts to ALL_CAPS.

**Comment by LyleSY** [comment]
Thanks, I haven't found any luck finding any body parts defined in the mod

**Comment by Coolthulhu** [comment]
> Thanks, I haven't found any luck finding any body parts defined in the mod

In mutations that grant armor. They have lists of parts that they cover. Some mutations also restrict equipment on some body parts.

**Comment by Coolthulhu** [comment]
Damn, the dino stats don't fuck around.
It's like a hulk town in there.

**Pull request merged**